akka-persistence/src/test/scala/akka/persistence/serialization/SerializerSpec.scala

```diff
@@ -221,6 +221,31 @@ class MessageSerializerPersistenceSpec extends AkkaSpec(customSerializers) {
       }
     }
 
+    "given PersistentRepr serialized with Akka 2.3.11 Scala 2.10" must {
```

Comment:
To be more realistic this probably should be Scala 2.11, since 2.4 is not released for 2.10. Worth the trouble you think?